packages/mint-components/src/components/tax-and-cash/sqm-tax-form/sqm-tax-document-submitted.feature

```diff
@@ -0,0 +1,57 @@
+@owner:andy
+@author:andy
+
+
+Feature: Tax Document Submitted View
+        View that informs the user of their Tax Document submission status.
+
+	Background: A user has submitted their Tax Document
+		Given a user is on the Tax Document Submitted view
+
+	@motivating
+	Scenario: View Tax Document Submitted
+		Given the user has submitted a tax document
+		When they view Document Submitted
+		Then they should see the status of their Tax Document and Banking Information
+
+	@minutia
+	@ui
+	Scenario Outline: The Tax Form header is controlled via <documentType> prop
+	Given the <documentType>
+	Then the Tax Form header will display "<documentType> Tax Form"
```

Comment:
If you can try to stay away from using "will". It ends up being simpler and easier to understand as you step through the spec

"Then the tax form header displays "<documentType> Tax Form"